### DISCUSSION TITLE
How to get the number of rows added and deleted by merge request？

### DISCUSSION THREAD

**Discussion by busy-tester** [python-gitlab/python-gitlab#2337] [Q&A] [answered]
As follows, I want to get the number of rows added and deleted by merge request

None of the following methods work

changes = mr.changes()、diffs = mr.diffs.list()、diff = mr.diffs.get(diff_id)

Please tell me what to do

![image](https://user-images.githubusercontent.com/43669330/197092412-5b850aae-a42a-4c35-b0e1-3f97befd71ac.png)

**Comment by JohnVillalovos** [comment, marked as answer]
Maybe look at the commits: https://docs.gitlab.com/ee/api/commits.html#get-a-single-commit

https://python-gitlab.readthedocs.io/en/main/gl_objects/commits.html

**Reply by busy-tester** [reply]
Thank you, can solve my needs
